packages/notices/src/store/actions.js

```diff
@@ -69,6 +71,7 @@ let uniqueId = 0;
 export function createNotice( status = DEFAULT_STATUS, content, options = {} ) {
 	const {
 		speak = true,
+		speakMessage = null,
```

Comment:
I don't get why we would need this - at least in this case. We shouldn't have two different messages.

Reply:
- The messages used for the `PreferenceToggleMenuItem` were different from the `label` used, which is why I implemented this
- If we remove this for the `Menu Items`, the message would be `Distraction Free off / Distraction Free on` which does not provide much context.
- Although for argument's sake, even `Distraction Free mode activated` also does not provide much context. 
- I would suggest expanding this custom message to give more context for e.g. : `Distraction Free view option is enabled/disabled for your editor`